src/explore-education-statistics-admin/src/pages/release/components/__tests__/PublicPreReleaseAccessForm.test.tsx

```diff
@@ -37,17 +37,6 @@ describe('PublicPreReleaseAccessForm', () => {
     expect(
       screen.getByText('Test person 2', { selector: 'li' }),
     ).toBeInTheDocument();
-
-    expect(
-      screen.getByText(
-        'http://localhost/publication/publication-1/release/release-1/prerelease',
```

Comment:
The assertions that have been removed should still exist for test suites related to the components where the links have been moved to.

Reply:
The link was moved to `ReleasePreReleaseAccessPage.tsx` which currently has no test suite. Since this branch has a load of tickets associated with it, I'll create a separate ticket to add a test suite for that component and pick that up next.

Reply:
Cool, fine by me :+1: